app/models/katello/host/content_facet.rb

```diff
@@ -18,11 +18,18 @@ class ContentFacet < Katello::Model
       has_many :applicable_rpms, :through => :content_facet_applicable_rpms, :class_name => "Katello::Rpm", :source => :rpm
       has_many :content_facet_applicable_rpms, :class_name => "Katello::ContentFacetApplicableRpm", :dependent => :destroy, :inverse_of => :content_facet
 
-      validates :content_view, :presence => true, :allow_blank => false
-      validates :lifecycle_environment, :presence => true, :allow_blank => false
+      validates :content_view, :presence => true, :allow_blank => false, :if => :can_use_content?
+      validates :lifecycle_environment, :presence => true, :allow_blank => false, :if => :can_use_content?
       validates :host, :presence => true, :allow_blank => false
       validates_with Validators::ContentViewEnvironmentValidator
 
+      def can_use_content?
+        host.operatingsystem.is_a?(::Redhat) &&
+          host.operatingsystem.kickstart_repos(host).first.present? &&
+          host.operatingsystem.name.downcase =~ /red/
+        # Other EL family systems don't need to use a content facet
```

Comment:
Don't need to, but can use a content facet correct? This comment and perhaps code seem misleading. I should be able to do the same things with a RHEL host as a CentOS host?